nixos/doc/manual/configuration/profiles/clone-config.xml

```diff
@@ -0,0 +1,11 @@
+
+<section xmlns="http://docbook.org/ns/docbook"
+         xmlns:xlink="http://www.w3.org/1999/xlink"
+         xmlns:xi="http://www.w3.org/2001/XInclude"
+         version="5.0"
+         xml:id="sec-profile-clone-config">
+ <title>Clone Config</title>
+ <para>
+  <emphasis>TBD</emphasis>
```

Comment:
```suggestion
  This profile is used in installer images.
  It provides configuration.nix that imports all the modules that
  were also used when creating the image in the first place.
  As a result it allow users to edit and rebuild the live-system.
```

Reply:
So that's the in-a-nutshell explanation, huh? Thanks!

Reply:
Yes. Otherwise rebuilding would result in not in the same system so adhoc updates cannot be applied.